packages/x-tailwindcss/CHANGELOG.md

```diff
@@ -1,16 +1,15 @@
 # Change Log
 
-All notable changes to this project will be documented in this file.
-See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file. See
```

Comment:
I guess changes in this file are not intended

Reply:
it was the linter, I dont know why but the file was wrong. check how the other versions has the See word in the first line